core/modules/path/tests/path_pattern.test

```diff
@@ -848,6 +864,27 @@ class PathPatternLocaleTestCase extends PathPatternFunctionalTestHelper {
     $this->assertEntityAlias('node', $node, 'french-node', 'fr');
     $this->assertAliasExists(array('pid' => $english_alias['pid'], 'alias' => 'content/english-node-1'));
   }
+
+  /**
+   * Language neutral German node has the correct transliteration applied.
+   */
+  public function testNeutralTransliteration() {
+    config_set('system.core', 'language_default', 'de');
+    config_set('path.settings', 'transliterate', TRUE);
+    // Note: "ü" -> "ue", "ß" -> "ss".
+    $title = 'Lückenbüßer';
```

Comment:
Could you save two nodes, both with this title? Or `Lückenbüßer DE` and `Lückenbüßer EN` if you want to load them by title. I think it's important to test that the exact same title is transliterated differently based on the site default language.

Reply:
That's a good idea.